packages/core/src/App/Containers/Layout/header/trading-hub-header.jsx

```diff
@@ -156,7 +156,7 @@ const TradingHubHeader = ({
                     <Divider />
                     <TradersHubHomeButton is_dark_mode={is_dark_mode} />
                 </DesktopWrapper>
-                <MenuLinks />
+                <MenuLinks is_traders_hub_routes />
```

Comment:
Where do you define this props?

Reply:
@mahdiyeh-deriv i assigned a default value for all other cases other than tradershub header.
We are passing value true  here